source/controllers/include/controllers/impedance/Impedance.hpp

```diff
@@ -35,7 +35,7 @@ class Impedance : Controller<S, S> {
    * @param feedback_state the real state of the system as read from feedback loop
    * @return the output command at the input state
    */
-  virtual S compute_command(const S& desired_state, const S& feedback_state) const;
+  virtual S compute_command(const S& desired_state, const S& feedback_state);
```

Comment:
Ah yes I noticed this right at the end of the previous PR, it was therefore hiding the base definition. Glad you caught it here.